src/model/schema.js

```diff
@@ -475,6 +470,70 @@ export default class Schema {
 		}, { priority: 'high' } );
 	}
 
+	/**
+	 * Registers custom properties to a given attribute.
```

Comment:
```suggestion
	 * Adds custom properties to an attribute.
```

Reply:
This was addressed by fix for one issues raised in previous review.